src/containers/App/App.js

```diff
@@ -78,12 +85,18 @@ const App = () => {
                 setNewPost={setNewPost}
               />
             </Route>
+            <Route exact path="/followers" render={() => {
+                return <Followers id={userID} />
+              }}/>
+            <Route exact path="/users/:id" render={({match}) => {
```

Comment:
I'm going to use this pathing from the search page, to lead users the profiles of who they've clicked on